### DISCUSSION TITLE
Feature Request - Add Snapfinity option

### DISCUSSION THREAD

**Discussion by wx-watch** [Le0Michine/FusionGridfinityGenerator#71] [Ideas]
Firstly, this is a fantastic add-on! Wondering if it would be possible to add a Snapfinity option to the baseplate, similar to the existing connection hole option. I am referring to this model and the little cutout at the bottom to allow for multiple baseplates to be connected via printed clips:

[https://www.printables.com/model/403457-gridfinity-snapfinity-baseplate](url)

Thank you!

**Comment by Le0Michine** [comment]
Hi, thanks for the feature request. Unfortunately at the moment I don't see how something like that could be done as there are a ton of different variation of this sort of modifications out there and it isn't feasible supporting them all
